docs/src/intro-java.md

```diff
@@ -59,7 +59,7 @@ public class Example {
     <dependency>
       <groupId>com.microsoft.playwright</groupId>
       <artifactId>playwright</artifactId>
-      <version>1.16.0</version>
+      <version>1.17.0</version>
```

Comment:
Looks unrelated

Reply:
it is a drive-by.

Reply:
Yes, but won't we be merging this into docs for 1.17?

Reply:
I don't think so that we will merge ToT docs with 1.17 docs. It would not work, since the ToT docs contain already too many unrelated changes.

Reply:
I was not suggesting to merge ToT docs into 1.17 docs. I was suggesting to keep this separate to remember that we need to merge it into 1.17.

Reply:
extracted into https://github.com/microsoft/playwright/pull/10609